spec/requests/waste_exemptions_engine/exemptions_forms_spec.rb

```diff
@@ -9,7 +9,7 @@ module WasteExemptionsEngine
     include_examples "GET form", :exemptions_form, "/exemptions"
     include_examples "go back", :exemptions_form, "/exemptions/back"
     include_examples "POST form", :exemptions_form, "/exemptions" do
-      let(:form_data) { { exemptions: Exemption.all.map(&:id).map(&:to_s) } }
+      let(:form_data) { { exemptions: create_list(:exemption, 10).map(&:id).map(&:to_s) } }
```

Comment:
Also wondering if the `before(:context) { create_list(:exemption, 5) }` on line 7 is affected or made redundant by this change.